tests/test_common/session_prefetcher.py

```
@@ -276,6 +276,55 @@ class _Built(NamedTuple):
+    ``abandon()`` only disconnects the parent side. With a rank already gone the
+    manager thread stays wedged in MPI, so the survivors are never told to exit
+    and keep their GPU memory while the replacement pool spawns on top of them.
+    Mirrors ``MpiPoolSession._teardown_unidentified_pool``, recycling guard
+    included.
+    """
+    import signal
+
+    try:
+        session.abandon()
+    except Exception:
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)